Controller/Index/Index.php

```diff
@@ -132,16 +136,15 @@ public function execute()
             } else {
                 throw new \Exception('Authentication failed from M3');
             }
-        }
-        catch (\Exception $e) {
+        } catch (\Exception $e) {
             $this->logger->writeLog($e->getMessage());
             $this->messageManager->addErrorMessage('Authentication failed');
         }
         $this->addAuthenticationCode($code);
         $this->_redirect($this->getRedirectPath());
     }
 
-    public function logincustomer($email)
+    public function loginCustomer($email)
```

Comment:
Please review the function. there are 2 functions named loginascustomer and logincustomer. Kindly change any one @NanthiniMuralidaran

Reply:
There are two functions - 
I just changed 'logincustomer' as 'login**C**ustomer' and I didn't change 'loginAsCustomer' since it has a good naming convention already.

@niranjanleanswift